File: Q4.cpp
#include <iostream>
#include <stdio.h>

using namespace std;

int main()
{
    int vezes = 1;
    for (int contador = 0; contador < 20; contador ++, vezes ++)
    {
        printf("%dª Vez: Eu gosto de estudar Algoritmos!\n", vezes);
    }
    return 0;
}

File: Q4.cs
using System;

namespace Q4
{
    class Program
    {
        static void Main(string[] args)
        {
            byte vezes = 1;
            for (byte contador = 0; contador < 20; contador ++, vezes ++)
            {
                Console.WriteLine("{0}ª Vez: Eu gosto de estudar Algoritmos!", vezes);
            }
        }
    }
}